tests/saitests/sai_qos_tests.py

```diff
@@ -222,8 +268,9 @@ def runTest(self):
             # dscp 48 -> queue 6
             # So for the 64 pkts sent the mapping should be -> 58 queue 1, and 1 for queue0, queue2, queue3, queue4, queue5, and queue6
             # Check results
-            assert(queue_results[QUEUE_0] == 1 + queue_results_base[QUEUE_0])
-            assert(queue_results[QUEUE_1] == 58 + queue_results_base[QUEUE_1])
+            assert(queue_results[QUEUE_0] >= 1 + queue_results_base[QUEUE_0])
+            # +1 to account for the pkt sent by get_rx_port()
+            assert(queue_results[QUEUE_1] == 58 + queue_results_base[QUEUE_1]) + 1
```

Comment:
Why is this +1 needed? queue_results_base is populated after sending the pkt in get_rx_port().

Reply:
Instead of adding a sleep, I just added +1 here. Just to be sure I'll move the read_port_counters before calling get_rx_port

Reply:
Please avoid the +1 here. since we are keeping the = for queue 0.

Reply:
It's still "==" comparison for queue1. Just wanted to avoid adding 10 sec delay, not a big deal. Removed +1.

Reply:
since we do >=0 for queue 0 and == 58 + 1 for queue 1, if there is an error and the packet in get_rx_port lands in queue 0 and packet for queue 0 lands in queue 1, we will still get a pass. To make it more reliable, I will probably increase the packet counts on each queue later. Thanks for fixing it

Reply:
I see. BTW DSCP for get_rx_port packet is fixed, and will always go to queue 1.